_posts/2018/02/2018-02-16-library-carpentry-governance.md

```diff
@@ -0,0 +1,83 @@
+day. People can then add extras from the [other modules under development](https://github.com/data-lessons) if they want to have 
+a two-day format, or 
+they can stretch the core curriculum to be taught over two days, which would give more time for people to practise 
+and embed what they have learned.
+
+### One day or two days?
+
+Librarians often struggle to be granted two consecutive days off for training, which is why the 
+one-day format might be the way we go forward for now. If we want to have a two-day format, 
+then the workshop could be split over four mornings, or we could teach single days in separate weeks.
+
+### Who is an instructor?
+
+The question of [Who is an instructor?](http://librarycarpentry.github.io/about) has been addressed 
+on the old website. Please add your thoughts. If we are to merge with the Carpentries, we need to move to a 
+model where instructors have completed the [Carpentries instructor training program](http://carpentries.github.io/instructor-training/). However, we do not want to mandate that certification for now as we do not have enough trained instructors. Belinda will ask the Carpentries to schedule two open trainings specifically for librarians this year.
+
+Though this was not raised at the meeting, we should consider developing an onboarding process for 
+certified Data and Software Carpentry instructors who would like to teach Library Carpentry.
+
+### Governance
+
+Belinda suggested that at the minimum we need a chair, a co-chair and a secretary, and suggested 
+she could be the liaison between the governance group and the Carpentries. Tim Dennis and Juliane Schneidler 
+have put together [this document](https://docs.google.com/document/d/1U0F4HCW7YcZx319_KL4d0i473C4sDx5P7jnwqejer9w/edit)
+for discussion about the role of a governance group.
+
+Greg Wilson proposed we review that document between now and May, with a view to electing an interim governance group
+when we meet at [CarpentryCon 2018](http://www.carpentrycon.org/) in Dublin. Please add your thoughts.
+
+### Workshop requests
+
+The Carpentries are now taking responsibility for organising Library Carpentry workshops. 
+[Use this form](https://software-carpentry.org/workshops/request/) to request a Library Carpentry workshop.
+
+### Library Carpentry Position based at CDL
+
+Interviews are ongoing for [this position](http://www.datacarpentry.org/jobs/). It was readvertised when no 
+appointment was made from the first round. We hope that there will be an appointment soon. The person in 
+the role will work jointly with CDL and the Carpentries, and will look at finalising lessons, and buiding a 
+network to grow workshop and instructor numbers in the US and more globally.
+
```

Comment:
Insert "Summing Up" heading (so that it's clear these following paragraphs aren't about the Lib Carp position)